moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -277,9 +285,10 @@ bool TrajectoryExecutionManager::push(const moveit_msgs::msg::RobotTrajectory& t
       for (const std::string& controller : context->controllers_)
         ss << controller << " ";
       ss << "]:" << std::endl;
-      for (const moveit_msgs::RobotTrajectory& trajectory_part : context->trajectory_parts_)
-        ss << trajectory_part << std::endl;
-      ROS_INFO_NAMED(name_, "%s", ss.str().c_str());
+      // TODO: Provide message serialization
```

Comment:
Please use a STREAM macro

Reply:
Unfortunately, the msg in ROS2 don't have a stream operator